tests/e2e/test_post_transaction.py

```diff
@@ -18,9 +18,7 @@ def setUp(self) -> None:
         self.test_safe = "0xAb4178341C37e2307726361eEAE47FCA606cd458"
         self.cow = Token("0x3430d04E42a722c5Ae52C5Bffbf1F230C2677600", 18)
         self.receiver = Address("0xFFcf8FDEE72ac11b5c542428B35EEF5769C409f0")
-        self.client = EthereumClient(
-            URI(f"https://goerli.infura.io/v3/{os.environ.get('INFURA_KEY')}")
-        )
+        self.client = EthereumClient(URI("https://rpc.ankr.com/eth_goerli"))
```

Comment:
Same as for the `test_token_details.py`, it should be `"https://rpc.ankr.com/eth_goerli"`.

I changed some code below to
```
        self.pk = "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
        self.test_safe = "0xAb4178341C37e2307726361eEAE47FCA606cd458" # this safe probably has to actually be deployed ...
        self.cow = Token("0x0625afb445c3b6b7b929342a04a22599fd5dbb59", 18)
        self.receiver = Address("0xFFcf8FDEE72ac11b5c542428B35EEF5769C409f0")
        self.client = EthereumClient(URI("https://rpc.ankr.com/eth_sepolia"))
```
but this fails. Probably since that safe is not actually deployed.

Reply:
Should we skip this test or find a deployed safe?

Reply:
https://github.com/cowprotocol/solver-rewards/issues/387